spring-cloud-stream/src/test/java/org/springframework/cloud/stream/binder/BinderFactoryConfigurationTests.java

```diff
@@ -90,6 +89,29 @@ public void loadBinderTypeRegistryWithOneBinder() throws Exception {
 	}
 
 	@Test
+	public void loadBinderTypeRegistryWithOneBinderAndSharedEnvironment() throws Exception {
```

Comment:
this test and `loadBinderTypeRegistryWithOneBinderAndIsolatedEnvironment` make the same assertions - shouldn't we be testing something that varies with the boolean property?